gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -700,6 +703,12 @@ public Builder setEndpoint(String endpoint) {
       return this;
     }
 
+    @InternalApi
```

Comment:
Can you please add some Javadocs (non-confidential) to this public method? So that other developers don't have to look up "what is hardBoundTokenTypes?".

Reply:
Done in https://github.com/googleapis/sdk-platform-java/pull/3467/commits/0e40f22636269b7db06281fe228cfb50dad2c0d5